chrome/content/zotero/elements/mergeGroup.js

```diff
@@ -142,7 +142,7 @@
 			this._mergePane.libraryID = this.libraryID;
 			this._leftPane.data = this._data.left;
 			this._rightPane.data = this._data.right;
-			this._mergePane.data = this._data.merge;
+			// Fix render error: Do not set data to _mergePane, since choosePane will update it
```

Comment:
If this isn't necessary it can just be removed — no need to leave an explanation.

Is the idea that we were always running choosePane anyway and this wasn't necessary, and now it causes some extra problem due to the asynchronous render?

Reply:
Yes. Not related to the async render, calling choosePane after setting data to the pane immediately would discard and create a new CE instance, which is unnecessary and throws a error from the CE initialization (because it’s discarded halfway of the initialization)